src/mumble/GlobalShortcut_win.h

```diff
@@ -129,6 +129,8 @@ class GlobalShortcutWin : public GlobalShortcutEngine {
 		/// @return      Returns true if the GlobalShortcut engine signalled that
 		///              the button should be suppressed. Returns false otherwise.
 		bool injectMouseMessage(MSG *msg);
+		private:
```

Comment:
A tab should be removed from the indentation, see the other access specifiers above.